src/components/MiniMenu/MiniMenu.scss

```diff
@@ -0,0 +1,92 @@
+@import "constants/style";
+
+.mini-menu {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  align-items: center;
+  gap: $spacing--xxs;
+  justify-content: center;
+
+  width: fit-content;
+  height: fit-content;
+  min-height: 48px;
+  padding-left: $spacing--xxs 2px;
+
+  border-radius: $rounded--full;
+  box-shadow: 0 0 20px rgba($navy--400, 0.12);
+
+  background-color: $gray--000;
+}
+.mini-menu__item {
+  all: unset;
```

Comment:
Why is it necessary here?

Reply:
to override the native styles of mini-menu__items